gui/wxpython/history/browser.py

```diff
@@ -226,11 +274,35 @@ def showCommandInfo(self, command_info):
     def clearCommandInfo(self):
         """Clear command info."""
         self.sizer_general_info.Clear(True)
-        self.sizer_region_settings.Clear(True)
+        self.sizer_region_settings_grid.Clear(True)
+        self.sizer_region_settings_text.Clear(True)
         self._createGeneralInfoBox()
         self._createRegionSettingsBox()
         self._layout()
 
+    def _get_current_region(self):
+        """Get current computational region settings."""
+        current_region = dict(
+            parse_key_val(gs.read_command("g.region", flags="g"), val_type=float)
+        )
```

Comment:
Why not `gs.region()`?

Reply:
I was wondering whether there is a better way.. Thanks Anna. I also simplified the similar code in history.py.